kong/db/schema/init.lua

```diff
@@ -81,6 +81,7 @@ local validation_errors = {
   REQUIRED                  = "required field missing",
   NO_FOREIGN_DEFAULT        = "will not generate a default value for a foreign field",
   UNKNOWN                   = "unknown field",
+  IMMUTABLE                 = "field cannot be updated",
```

Comment:
We should state that immutability is the reason why the field cannot be updated to be consistent with other error messages self identifying.

Reply:
👍